docs/pages/middleware.rst

```diff
@@ -163,7 +163,8 @@ Use with ``wrap_middleware``:
 
 .. literalinclude:: /examples/middleware/rate_limit.py
   :linenos:
-  :lines: 33-
+  :lines: 33-49
+  :no-run:
```

Comment:
If we don't run it, maybe let's remove the command instead? Why don't we run it though?